apps/ewallet/test/ewallet/web/paginator_test.exs

```diff
@@ -55,6 +55,24 @@ defmodule EWallet.Web.PaginatorTest do
       assert paginator.pagination.per_page == 4
     end
 
+    test "returns a paginator with the given `page_record_value` without `page_record_field`" do
+      ensure_num_records(Account, 1)
+
+      id = from(a in Account, select: a.id, order_by: a.id)
+
+      id =
+        id
+        |> Repo.all()
+        |> Enum.at(0)
+
+      paginator =
+        Paginator.paginate_attrs(Account, %{"page_record_value" => id, "per_page" => "5"}, [:id])
+
+      assert paginator.pagination.current_page == 1
+      assert paginator.pagination.per_page == 5
+      assert paginator.pagination.count == 1
+    end
+
```

Comment:
Can you add another test that tries another field type e.g. `created_at` (this will involve field name mapping too)?